src/harshadBJun20/Assignment22/CheckNumberPalindromeOrNot.java

```diff
@@ -0,0 +1,32 @@
+package harshadBJun20.Assignment22;
+/*Assignment 22 : 1st Jul 2020
+
+Program 1: 
+Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+*/
+public class CheckNumberPalindromeOrNot {
+	
+	void checkPalindrome(int num) {
+		int seperateNum=0, reversedNum = 0;
```

Comment:
No need to use extra variable seperateNum.

Reply:
ok